src/main/java/io/sprucehill/zalando/api/service/CategoryService.java

```diff
@@ -0,0 +1,41 @@
+package io.sprucehill.zalando.api.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.methods.HttpGet;
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import io.sprucehill.zalando.api.exception.NotFoundException;
+import io.sprucehill.zalando.api.model.Category;
+import io.sprucehill.zalando.api.model.Domain;
+
+public class CategoryService extends AbstractService implements ICategoryService{
+
+	@Override
+	public List<Category> list(Domain domain) throws NotFoundException{
+		HttpGet request = getRequest("/categories");
+		request.addHeader(HttpHeaders.ACCEPT_LANGUAGE,domain.getLocale());
+		return execute(request, new TypeReference<List<Category>>() {});
+	}
+
+	@Override
+	public List<Category> list(Domain domain, String targetGroup,String[] keys) throws NotFoundException {
```

Comment:
The implementation of this methods basically seems to only wrap a for each loop.

I'd suggest to either implement a 'real' multi get by spawing multiple requests or simply dispose of this method